src/frontend/js/components/CourseGlimpse/index.tsx

```diff
@@ -43,6 +48,16 @@ const CourseGlimpseBase = ({ context, course }: CourseGlimpseProps & CommonDataP
         </div>
       ) : null}
       <div className="course-glimpse__wrapper">
+        <p className="course-glimpse__code">
+          {course.code ? (
+            <FormattedMessage
+              {...messages.ref}
+              values={{
+                code: course.code,
+              }}
+            />
+          ) : null}
```

Comment:
Just a shorcut which improves readability from my point of view.
```suggestion
          {course.code &&
            <FormattedMessage
              {...messages.ref}
              values={{
                code: course.code,
              }}
            />
          }
```

Reply:
IMO the ternary is clearer and almost as short as the `&&` to have conditionals in JSX. [Kent C. Dodds](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) agrees with me, although I'll admit it's more about readability and the error cases he mentions are a rare occurence.

Reply:
+1 to keep the ternary in this PR as it is what is used in the code base. If we want to switch to `&&` there can be a PR for this on the whole code base :wink:

Reply:
Ok let's keep this syntax 👍
I note to myself to use it elsewhere in the codebase.